src/H5Ipkg.h

```diff
@@ -307,11 +314,20 @@ typedef struct H5I_suint64_t {
  *      type info free list length.  This is of necessity a soft limit as entries cannot
  *      be removed from the head of the free list unless they are re-allocable.
  *
- * num_type_info_fl_entries_reallocable:  Atomic uint64_t containing the number of entries 
- *      in the type info free list that are known to have no remaining threads in H5I
- *      that retain pointers to them, and are thus reallocable.  If this field is positive, 
- *      any thread may decrement it, and then take the next entry off the head of the type
- *      info list and re-use it.
+ * type_next_sn: Atomic uint64_t containing the serial number for the next entry on the 
+ *      type info free list. Whenever an entry is added to the id info free list, that 
```

Comment:
Comment for `type_next_sn` refers to id info and id variables instead of type info and type variables